meilisearch/index.py

```diff
@@ -2042,6 +2042,68 @@ def reset_proximity_precision(self) -> TaskInfo:
 
         return TaskInfo(**task)
 
+    # LOCALIZED ATTRIBUTES SETTINGS
+
+    def get_localized_attributes(self) -> Union[List[Dict[str, List[str]]], None]:
```

Comment:
> Not very experienced with python, let me know if I should have created a model for the localized attributes, similar to embedders or proximity precision.

Let's do that, it will make it easier to work with. So you will have:

```py
class LocalizedAttributes(CamelBase):
    ...
```

```suggestion
    def get_localized_attributes(self) -> Union[LocalizedAttributes, None]:
```

Reply:
Excellent question! You are correct this should have been a list. I updated my suggestion for this.